config/config.yaml

```diff
@@ -1,10 +1,17 @@
 # This is the default config file, you can override it by creating a config.yaml in the same directory
 # by default, all linters are enabled if you don't specify. You can disable them by setting enable to false
 # example1: disable staticcheck for org
-# qbox:
+qbox:
```

Comment:
The default behavior should be fixed in code, no need to config in this. Principle is "the fewer configurations, the better."